libc/src/strings/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+function(add_bcmp bcmp_name)
+  add_implementation(bcmp ${bcmp_name}
+    SRCS ${LIBC_SOURCE_DIR}/src/strings/bcmp.cpp
+    HDRS ${LIBC_SOURCE_DIR}/src/strings/bcmp.h
+    DEPENDS
+      libc.include.string
```

Comment:
should we also add `libc.include.strings`?
Here and below.

Reply:
Should I just remove having a dependency on our includes? Other functions like `bcopy`, `index`, etc, below don't have such dependencies.  `bcmp` and `bzero` don't explicitly include anything from `libc/include/`.

30655b4a4336

Reply:
Sure let's remove them then.